src/commands/tasks/util.ts

```diff
@@ -25,3 +30,154 @@ export async function zipFileFromDirectory(path: string): Promise<string> {
     archive.finalize();
   });
 }
+
+export async function createFunctionUser(sdk: OAuth1Client, data: { taskName: string; targetEmail?: string; targetPermissions: string[]; }) {
+  const { taskName, targetEmail, targetPermissions } = data;
+
+  const email = targetEmail || `exh.tasks+${taskName}@extrahorizon.com`;
+  validateEmail(email);
+
+  const password = `A-${uuidv4()}-z`;
```

Comment:
(minor) Can we also add a number to the base password and a comment we're doing this to satisfy a broad range of possible password policies?